NewsApp/NewsApp/Controller/NewsController.swift

```diff
@@ -11,9 +11,10 @@ import UIKit
 class NewsController: UITableViewController {
     private let apiClient = APIClient()
     private var newsArticles: [Article] = []
+    var category: Category?
```

Comment:
Je connais pas vraiment le contexte mais cette variable n'est jamais setter puis tu acceptes d'envoyer un nil dans ton call api ?

Reply:
Oui elle est set depuis une autre classe, uniquement si on choisit une catégorie particulière

Reply:
fix in "refacto viewController"